Cargo.toml

```diff
@@ -30,7 +30,11 @@ lazy_static = "1.3.0"
 windows = { version = "0.51.1", optional = true, features = [
     "Win32_Graphics_Direct3D11",
 ] }
+enumset = "1.1.10"
+nalgebra = { version = "0.33.2", optional = true }
+
 
 [features]
 default = []
 submit_d3d11 = ["dep:windows"]
+nalgebra=["dep:nalgebra"]
```

Comment:
Where exactly is this dep used?

At LIV the library we prefer to handle vector/matrix maths is [glam](https://crates.io/crates/glam)

I do not know how popular `nalgebra` is, nor how relevent it is to add an optional feature to add bindings/conversion for its types to this crate.

Obtaining a flat array/structure of numbers is probably the best way to get the data, if not we would need to start supporting every other mathematics library around?